src/pages/settings/organization/Credits.vue

```
@@ -0,0 +1,866 @@
+                    <div class="font-semibold text-gray-900 dark:text-white">
+                      Grants & purchases
+                    </div>
+                  </td>
+                  <td
+                    class="whitespace-nowrap px-4 py-3 text-right font-semibold text-emerald-600 dark:text-emerald-400"
+                  >
+                    +{{ formatCredits(day.grantsTotal) }}
+                  </td>
+                </tr>
+                <tr
+                  v-for="(entry, idx) in metricsWithData(day)"
```

ESLint

[error] 'idx' is defined but never used.

(vue/no-unused-vars)